conf/config.toml

```diff
@@ -20,8 +20,21 @@ tso-save-interval = "3s"
 [log]
 level = "info"
 
-#[log.file]
+# log format, one of json, text, console
+#format = "text"
```

Comment:
em, I think most of the time, users don't use console even they run PD in the console. TEXT and JSON may be enough.